at.ssh-mitm.server.json

```diff
@@ -0,0 +1,138 @@
+{
+    "app-id": "org.flatpak.Hello",
```

Comment:
definitely the wrong app id.

Reply:
I have changed to app-id  to "at.sshmitm.server".

My main domain is "ssh-mitm.at" but the dash is not allowed in the domain part :-(
This is the reason, why I had to use "at.sshmitm.server". 
Note: this domain belongs to me.

Reply:
it should be `at.ssh_mitm.server` then. the rules is to replace a `-` in the domain part with `_` to avoid ambiguities, since `_` is invalid in DNS.